src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -81,11 +85,18 @@ def plot_persistence_barcode(
     :param axes: A matplotlib-like subplot axes. If None, the plot is drawn on
         a new set of axes.
     :type axes: `matplotlib.axes.Axes`
+    :param fontsize: Fontsize to use in axis.
+    :type fontsize: int
+    :param title: title for the plot.
+    :type title: string
```

Comment:
Title was one of the reason why I was returning the `axes` (the other reason was for subplotting). One can do:
```python
import matplotlib.pyplot as plot
import gudhi

persistence_file= '/data/persistence_diagram/rips_on_tore3D_1307.pers'
ax = gudhi.plot_persistence_diagram(persistence_file=persistence_file, legend=True)
ax.set_title('pouet')
plot.show()
```
I am ok for both solutions, but I think there are already a lot of arguments for these methods. Maybe we could just document this.

Reply:
Ah, I didn't noticed `ax` was returned ; it is not straightforward when looking at the user manual. Indeed it seems natural to go the way you suggest, and remove `title` as an argument of the function (I agree it is a bit overloaded).